plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -516,6 +519,11 @@ class producer_plugin_impl : public std::enable_shared_from_this<producer_plugin
                           ("txid", trx->id())
```

Comment:
I know you called this `transaction_trace_dump` but for the case of an invalid signature, probably should log the transaction.

Reply:
Actually everywhere we currently log REJECTING or ACCEPTING we should log to this logger with either the trace or the transaction.

Reply:
Fixed.